kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -338,7 +343,7 @@ private void removeClassFromJaxbContextClassList( List<Class> classList, Illegal
                String className = classLocs.listIterator(classLocs.size()).previous().toString();
                Class removeClass = null;
                try {
-                   removeClass = Class.forName(className);
+                   removeClass = projectClassLoader.loadClass(className);
```

Comment:
Personally, I prefer `ClassLoader.loadClass(className)` simply because I know exactly what is going on there: the `ClassLoader.loadClass(className)` method is directly calling the Project ClassLoader. 

I'm pretty sure that the class does not have to be initialized (seeing as the project classloader has already loaded it). 

Class.forName(clasName, init, classLoader) is doing security checks which we don't really need.